addons/func_godot/src/trenchbroom/trenchbroom_game_config.gd

```diff
@@ -86,44 +100,7 @@ func build_class_text() -> String:
 	var brushface_tags_str = parse_tags(brushface_tags)
 	var uv_scale_str = parse_default_uv_scale(default_uv_scale)
 	
-	var config_text : String = """{
-	"version": 8,
-	"name": "%s",
-	"icon": "icon.png",
-	"fileformats": [
-		%s
-	],
-	"filesystem": {
-		"searchpath": ".",
-		"packageformat": { "extension": ".zip", "format": "zip" }
-	},
-	"textures": {
-		"root": "textures",
-		"extensions": [".bmp", ".exr", ".hdr", ".jpeg", ".jpg", ".png", ".tga", ".webp"],
-		"excludes": [ %s ]
-	},
-	"entities": {
-		"definitions": [ %s ],
-		"defaultcolor": "0.6 0.6 0.6 1.0",
-		"scale": %s
-	},
-	"tags": {
-		"brush": [
-			%s
-		],
-		"brushface": [
-			%s
-		]
-	},
-	"faceattribs": { 
-		"defaults": {
-			%s
-		},
-		"contentflags": [],
-		"surfaceflags": []
-	}
-}
-"""
+	var config_text : String = get_compatible_game_config_text()
```

Comment:
`get_compatible_game_config_text()` is superfluous indirection. Instead this could be where the match statement I recommended earlier would go:
```gdscript
	var config_text : String = ""
	match game_config_version:
		GameConfigVersion.Latest:
			config_text = get_game_config_v8_text()
		GameConfigVersion.Version4:
			config_text = get_game_config_v4_text()
		GameConfigVersion.Version8:
			config_text = get_game_config_v8_text()
		_:
			push_error("Unsupported Game Config Version!")